packages/flutter_driver/lib/src/find.dart

```diff
@@ -26,7 +26,7 @@ abstract class CommandWithTarget extends Command {
   }
 
   /// Deserializes the command from JSON generated by [serialize].
-  CommandWithTarget.deserialize(Map<String, String> json)
```

Comment:
i believe these changes are wrong. The documentation and variable names are wrong also. This isn't actually JSON. It's service-protocol-map-data, which is string->string. @yjbanov may be able to clarify further.